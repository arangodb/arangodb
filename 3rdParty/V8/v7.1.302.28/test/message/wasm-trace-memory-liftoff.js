// Copyright 2017 the V8 project authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

// Flags: --no-stress-opt --trace-wasm-memory --liftoff --no-future
// Flags: --no-wasm-tier-up

load("test/message/wasm-trace-memory.js");
