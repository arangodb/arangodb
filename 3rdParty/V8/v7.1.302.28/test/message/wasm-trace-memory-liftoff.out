liftoff     func:     0+0x3     load from 00000004 val: i32:0 / 00000000
liftoff     func:     1+0x3     load from 00000001 val:  i8:0 / 00
liftoff     func:     3+0x5      store to 00000004 val: i32:305419896 / 12345678
liftoff     func:     0+0x3     load from 00000002 val: i32:1450704896 / 56780000
liftoff     func:     1+0x3     load from 00000006 val:  i8:52 / 34
liftoff     func:     2+0x3     load from 00000002 val: f32:68169720922112.000000 / 56780000
liftoff     func:     4+0x5      store to 00000004 val:  i8:171 / ab
liftoff     func:     0+0x3     load from 00000002 val: i32:1454047232 / 56ab0000
liftoff     func:     2+0x3     load from 00000002 val: f32:94008244174848.000000 / 56ab0000
